sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,393 @@
+- Added new overloads and missing stride restrictions to [`sycl_ext_intel_matrix`](https://github.com/intel/llvm/blob/475ca2d0af8de9cf8622e27e33ae241391d85ed6/sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_intel_matrix.asciidoc) extension for offset load/store operations. intel/llvm#15499
+- Updated [EnvironmentVariables.md](https://github.com/intel/llvm/blob/13951ed7950b65f89ed1c07cc5fd1f023f92a8ce/sycl/doc/EnvironmentVariables.md) to clarify environment variables related to the program cache. intel/llvm#15795
+- Added [`GetStartedGuide.md`](https://github.com/intel/llvm/blob/1c1a9646be5302d42e02380e90eb7f5172322be2/sycl/doc/GetStartedGuide.md) with instructions to build `DPC++` for ARM processors. intel/llvm#15325
+- Added `indeterminate` to [`sycl_ext_oneapi_work_group_memory`](https://github.com/intel/llvm/blob/acf2ca8edcd62511e7cc6f6e49554e71931e7cbe/sycl/doc/extensions/proposed/sycl_ext_oneapi_work_group_memory.asciidoc) extension. The default constructor for `work_group_memory` now requires the `indeterminate` parameter to indicate it is a dummy object and must be assigned before use. intel/llvm#15933
+- Updated [`sycl_ext_oneapi_peer_access`](https://github.com/intel/llvm/blob/4950917ae83b0f596d33646b3c7fe6fa41c68b2d/sycl/doc/extensions/supported/sycl_ext_oneapi_peer_access.asciidoc) extension to indicate that P2P memory access is supported for HIP and Level Zero as well. intel/llvm#15847
+- Updated documentation to reflect the Plugin Interface removal. intel/llvm#15057
+- A new SYCL graph enqueue function, `execute_graph`, has been added to the  [`sycl_ext_oneapi_enqueue_functions`](https://github.com/intel/llvm/blob/66867d4faf87e03b855e3dc3d004f6b39c7553cd/sycl/doc/extensions/experimental/sycl_ext_oneapi_enqueue_functions.asciidoc) extension and [`sycl_ext_oneapi_graph`](https://github.com/intel/llvm/blob/66867d4faf87e03b855e3dc3d004f6b39c7553cd/sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc) extension has been updated accordingly. intel/llvm#15677
+- Updated [`KernelProgramCache.md`](https://github.com/intel/llvm/blob/e127a2e913d5a357095a12408cbff27cc2a64dfa/sycl/doc/design/KernelProgramCache.md) with details on in-memory eviction. intel/llvm#16129
+- Enhanced [sycl_ext_oneapi_in_order_queue_events](https://github.com/intel/llvm/blob/65849fdc063eb1ca5a77cfed759f5a8c4856e413/sycl/doc/extensions/experimental/sycl_ext_oneapi_in_order_queue_events.asciidoc) extension to allow `ext_oneapi_get_last_event` on queues with discarded events. intel/llvm#15638
+- [`sycl_ext_oneapi_bindless_images`](https://github.com/intel/llvm/blob/f322f232c629600d85bec56f05d5979b2d61e438/sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc) extension was bumped to revision 6 to reflect the changes since revision 5. intel/llvm#14953
+
+### SYCLcompat
+
+- Enabled `device_info` caching within `device_ext`. intel/llvm#14630
+- Bumped `SYCLCOMPAT_COMPATIBILITY_TEMP` to 900. intel/llvm#15375
```

Comment:
@intel/syclcompat-lib-reviewers Could you please clarify what is the meaning of `SYCLCOMPAT_COMPATIBILITY_TEMP` and whether it should be mentioned in release notes. I am not sure if it is an internal detail or indicates the version of compat library.